pages/index.js

```diff
@@ -71,42 +71,50 @@ export default function Home() {
 					vs_currencies: ['usd'],
 			});
 
-			setPrice(data.vechain.usd)
+			setVetPrice(data.vechain.usd)
 		}
 
     getPrice()
 	}, [])
 
 	useEffect(() => {
-    const getData = () => {
+      if (!vetPrice || tokens.length > 0) return;
       const promises = allTokens.map(async (item) => {
         const { data } = await axios.get(`/api/${item.address}`)
         item = { ...data, ...item }
 
+        // Multiplied by two because it's a 50-50 pool
+        item.tvlInUsd = item.reserves * item.price.base2quote * vetPrice * 2;
```

Comment:
you can factor our `vetPrice` because you multiply both the numerator and denominator by `vetPrice`
```suggestion
        item.tvlInVet = item.reserves * item.price.base2quote * 2;
```

Reply:
If you want usd because you are using it on the display, that's fine - happy for  you to ignore these suggestions

Reply:
yeah since everything is in USD now, let's leave it as that